.gitignore

```
@@ -2,6 +2,7 @@
 __pycache__/
 *.py[cod]
 *$py.class
-
+.idea
+.venv
 # C extensions
-*.so
\ No newline at end of file
+*.so
```

We could add here an ignore for all *.sqlite files.
As the Database is created with code we don't need to track it with Git
```suggestion
*.so
*.sqlite
```

---

We should untrack the .idea and sqlite files as well. I can show you how to do it tomorrow.